spec/features/venues/view_venue_details_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'View details of the venue' do
+  let!(:venue) { FactoryGirl.create(:venue) }
+
+  scenario 'When a user clicks on the venue name see venue details.' do
+    visit venues_path
+
+    click_link(venue.name)
+
+    expect(page).to have_content(venue.name)
+  end
+
+  scenario 'When user vists page should see venue details' do
```

Comment:
Look at this sentence a little more. Could you make it more explicit? For example 'when user visits page' -- what page? The index page? The show page?